### ISSUE TITLE
Bad message from Tesla API stream: <html>

### ISSUE THREAD

**Issue by kodek** [kodek/tesler#1]
E0420 12:41:36.303443       8 car_info_pub.go:70] Error fetching vin XXXX. Retrying in (1m22.165s): Bad message from Tesla API stream: <html>

I think the stream is no longer relevant. It doesn't work in the Model 3 anyway. Let's remove it.

**Comment by kodek** [comment]
#d68163a028e02df0df650f58cd5190f0e63314bf should hopefully unblock the issue.
